src/form-group.scss

```diff
@@ -21,13 +21,26 @@ $block: #{$fd-namespace}-form-group;
   &--inline {
     display: flex;
     justify-content: flex-start;
+    margin-right: 1rem;
```

Comment:
value should be moved to variable since its used in rtl mode as well

Reply:
Also visual tests are failed

Reply:
I decided to remove this margin, it's leftover after old `forms`. Now it's used only to make inline checkboxes/radio, which has labels with paddings/margins.